lib/src/impl_stub/impl_stub.dart

```diff
@@ -0,0 +1,14 @@
+library webcrypto.impl_stub;
+
+import 'package:webcrypto/src/impl_interface/impl_interface.dart';
+
+part 'impl_stub.aescbc.dart';
+
+const webCryptImpl = _WebCryptoImpl();
```

Comment:
Again, not that it really matters, but now that static interface to each library is exactly the same :rofl: 
```suggestion
const WebCryptoImpl webCryptImpl = _WebCryptoImpl();
```